lib/events/api.go

```diff
@@ -799,10 +799,14 @@ type UploadMetadataGetter interface {
 	GetUploadMetadata(sid session.ID) UploadMetadata
 }
 
-// StreamWriter implements io.Writer to be plugged into the multi-writer
-// associated with every session. It forwards session stream to the audit log
-type StreamWriter interface {
+// SessionRecorder records session events. It can be used both as a
+// [io.Writer] when recording raw session data and as a [apievents.Recorder]
+// when recording session events.
+type SessionRecorder interface {
```

Comment:
So we have both an `events.Recorder` interface and an `events.SessionRecorder` interface in this package, and they both are used for session recording? That's a bit confusing IMO.

Reply:
Yeah that's a good point, I don't need both those interfaces now. Removed `Recorder` and `StreamManager`.